ptvsd/wrapper.py

```diff
@@ -785,19 +799,20 @@ def process_launch_arguments(self):
         if self.launch_arguments.get('fixFilePathCase', False):
             self.path_casing.enable()
 
-        if self.launch_arguments.get('redirectOutput', False):
+        if self.launch_arguments.get('redirectOutput', False) \
+            or self.debug_options.get('REDIRECT_OUTPUT', False):
             redirect_output = 'STDOUT\tSTDERR'
         else:
             redirect_output = ''
         self.pydevd_request(pydevd_comm.CMD_REDIRECT_OUTPUT, redirect_output)
 
-    def __parse_launch_option_value(self, value):
+    def __parse_debug_option_value(self, value):
         if value == 'True' or value == 'False':
```

Comment:
I didn't notice this before, but that seems dangerous - what if we have a string option that just happens to be True or False? I think it's better to make a dict that maps options to their types/parsers, like so:
```py
DEBUG_OPTIONS = dict(REDIRECT_OUTPUT=bool, INTERPRETER_OPTIONS=unquote, ...)
```
Then when you're parsing the options, you can iterate over that dict to see which ones you care about, and invoke the function that's needed for that particular option.